feature-libs/cart/base/components/cart-coupon/cart-coupon.component.html

```diff
@@ -7,34 +7,58 @@
     <form (ngSubmit)="applyVoucher()" [formGroup]="couponForm">
       <label class="cx-cart-coupon-title">
         {{ 'voucher.coupon' | cxTranslate }}
-        <abbr
-          *cxFeature="'a11yRequiredAsterisks'"
-          class="text-decoration-none"
-          title="{{ 'common.required' | cxTranslate }}"
-          >*</abbr
-        >
+        <ng-container *cxFeature="'!a11yDisabledButtonInsteadOfRequiredFields'">
```

Comment:
I think we should have a TODO comment here with the ticket number for breaking change cleanup later.

Reply:
Changed